config/cable.yml

```diff
@@ -1,11 +1,8 @@
 development:
-  adapter: redis
-  url: redis://localhost:6379/1
+  adapter: enhanced_postgresql
 
 test:
-  adapter: test
+  adapter: enhanced_postgresql
 
 production:
```

Comment:
V produkcii sa odporuca to bezat standalone, ale nemyslim si ze tu bude nejaky velky traffic ze by nas to malo trapit. https://guides.rubyonrails.org/action_cable_overview.html#running-standalone-cable-servers cc @celuchmarek co myslis?